examples/pcf8523_monitor_timer/pcf8523_monitor_timer.ino

```diff
@@ -0,0 +1,55 @@
+/**
+ * Sets a timer for 10 seconds, and watches a pin (MONITOR_PIN) for
+ * its interrupt signal.
+ *
+ * The RTC's INT line is pulled down when the timer goes off and the
+ * interrupt is active.
+ */
+#include <Wire.h>
+#include "RTClib.h"
+
+#define MONITOR_PIN 5
+
+RTC_PCF8523 rtc;
+
+void setup () {
+  Serial.begin(9600);
+  if (! rtc.begin()) {
+    Serial.println("Couldn't find RTC");
+    while (1);
```

Comment:
```suggestion
    Serial.flush();
    abort();
```
This is for consistency with the other examples and also, more importantly, in order to avoid [undefined behavior][ub].

[ub]: https://en.cppreference.com/w/cpp/language/ub#Infinite_loop_without_side-effects